ghost/core/core/server/data/migrations/versions/6.0/2025-06-23-10-03-26-members-nullable-uuid.js

```diff
@@ -0,0 +1,4 @@
+const {createDropNullableMigration} = require('../../utils');
+
+// We need to disable foreign key checks because if MySQL is missing the STRICT_TRANS_TABLES mode, we cannot revert the migration
+module.exports = createDropNullableMigration('members', 'uuid', {disableForeignKeyChecks: true});
```

Comment:
The `disableForeignKeyChecks` option only applies to the `up` no the `down` - so I think the comment is incorrect.

The `uuid` column isn't used as a foreign key anywhere, so do we need this flag at all?

As far as I know the 6.0 migrations are irreversible, so we don't need to provide a `down` or have the ability to revert this migration 🤔 Maybe we want to be explicit with that?

```ts
const migration = createDropNullableMigration('members', 'uuid');

module.exports = {
  ...migration,
  config: {
    ...migration.config,
    reversible: false,
  },
  async down() {
    throw new Error('Irreversible migration');
  }
};
```